_topic_map.yml

```diff
@@ -331,7 +331,38 @@ Topics:
 - Name: Disabling the web console
   File: disabling-web-console
   Distros: openshift-enterprise,openshift-webscale,openshift-origin
-
+---
+Name: Security
+Dir: security
+Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
```

Comment:
Okay. I'll check with Vikram.

Reply:
I would take this off dedicated. There are no guarantees that this would apply there.

@lamek @sfortner-RH

Reply:
I spoke to @lamek and he suggested adding openshift-aro to the distros as well. There is some content he will want to adapt for Azure, but he said it can wait until after this PR is merged. So, for the time being, I have added this Distros line to each section:
Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-aro